table/block_based/block_based_filter_block.cc

```diff
@@ -157,9 +158,8 @@ void BlockBasedFilterBlockBuilder::GenerateFilter() {
 
   // Generate filter for current set of keys and append to result_.
   filter_offsets_.push_back(static_cast<uint32_t>(result_.size()));
-  BloomFilterPolicy::CreateFilter(tmp_entries_.data(),
-                                  static_cast<int>(num_entries), bits_per_key_,
-                                  &result_);
+  DeprecatedBlockBasedBloomFilterPolicy::CreateFilter(
```

Comment:
Why is this "Deprecated" rather than "BloomLike"?

Reply:
This code path and `CreateFilter` are exclusive to the deprecated block-based filter. See #9535

Reply:
This is hard to tell from the code.  I can see where GenerateFilter is called in multiple places.  How can you tell the code is dead?  Is this entire file mainly dead then? 

Can you add some comments somewhere about the deprecated state of these functions?

Reply:
Read HISTORY.md, "any built-in FilterPolicy can still read block-based filters." The code is not dead. Dead code should be removed.

Reply:
Added some clarification